### PR TITLE
Fix Currency field Issues - Edit and alignment

### PR DESCRIPTION
**Pull request by yonihorwitz**
## Related Issue
[#534](https://github.com/mbrn/material-table/issues/534) - Editable Currency type render "ok"

## Description
When editing a currency field, instead of returning "ok", show a right-aligned textField.  Also, right align currency field render when not in Edit mode.

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Affects MTableEditField and MTableCell

## Additional Notes
I attempted a previous PR for the same issue earlier - this is a new and improved version.  Still though, I'm new to working with GitHub so would appreciate any feedback.

### CODE REVIEW

File: src/components/m-table-edit-field.js [hunk 1]
```diff
@@ -123,7 +123,20 @@ class MTableEditField extends React.Component {
...
+    return (
+      <TextField
+        {...this.getProps()}
+        placeholder={this.props.columnDef.title}
+        value={this.props.value === undefined ? '' : this.props.value}
+        onChange={event => this.props.onChange(event.target.value)}
+        inputProps={{
+          style: {
+            fontSize: 13,
+            textAlign: "right"
```
Comment (eddiemonge) [inline src/components/m-table-edit-field.js]: should copy the `style={{ float: 'right' }}` like from TextField above

File: src/components/m-table-edit-field.js [hunk 2]
```diff
@@ -123,7 +123,20 @@ class MTableEditField extends React.Component {
   }
 
   renderCurrencyField() {
-    return "ok";
+    return (
+      <TextField
+        {...this.getProps()}
+        placeholder={this.props.columnDef.title}
+        value={this.props.value === undefined ? '' : this.props.value}
```
Comment (eddiemonge) [inline src/components/m-table-edit-field.js]: missing `type='number'`

**Comment by mforman1** [comment]
Hi,
Any update on the above? Can that be merged?
Thanks.

**Comment by mforman1** [comment]
Thanks! Any idea when the next version will be released?

**Pull request merged**